packages/sensors_plus/tizen/src/device_sensor.cc

```diff
@@ -67,7 +67,7 @@ bool DeviceSensor::StartListen(SensorEventCallback callback) {
   }
```

Comment:
For consistency, we should either move the `sensor_create_listener` call (and also `sensor_get_default_sensor`) to the constructor, or destroy the listener in `DeviceSensor::StopListen`. (Personally I prefer the former.)

Reply:
done